### PR TITLE
feat: add validation script

### PR DESCRIPTION
**Pull request by scottames**
Adds simple script to validate the cross-account and spark roles (+subnets for emr) are configured correctly in the target data plane account

### CODE REVIEW

File: scripts/validate.py [hunk 1]
```diff
@@ -0,0 +1,382 @@
+#!/bin/env python3
```
Comment (jakewhlr) [inline scripts/validate.py:1]:
```suggestion
#!/usr/bin/env python3
```

File: scripts/validate.py [hunk 2]
```diff
@@ -0,0 +1,382 @@
...
+
+import argparse
+import json
+import os
+import requests
+import time
+
+import jinja2
+import boto3
```
Comment (jakewhlr) [inline scripts/validate.py:10]: Can we add a `requirements.txt` for the reqs?

File: scripts/README.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
+# Tecton Terraform Setup Scripts
+
+## Validate
+
+Databricks example:
+
+- Assuming the default aws credentials can assume the ca-role
+    - If necessary, use of `aws-vault exec <some_other_role> -- ...` may be done as well for
+      role chaining
+
```
Comment (jakewhlr) [inline scripts/README.md:10]: I think we should be clear about the permissions needed to run this, should this be run as the role we are testing, or an admin role for the aws account?

File: scripts/validate.py
Comment (jakewhlr) [inline scripts/validate.py:1]: Can we name this a bit more specific? `validate-tecton-roles.py` or something?

Comment (scottames) [reply scripts/validate.py:1]: Addressed in https://github.com/tecton-ai-ext/tecton-terraform-setup/pull/103/commits/2c6193c60517acb97f7eb69d05fcfc5d11c4bf03

File: scripts/README.md (hunk 3 above)
Comment (scottames) [reply scripts/README.md:11]: Addressed in https://github.com/tecton-ai-ext/tecton-terraform-setup/pull/103/commits/2c6193c60517acb97f7eb69d05fcfc5d11c4bf03

File: scripts/validate.py (hunk 2 above)
Comment (scottames) [reply scripts/validate.py]: Addressed in https://github.com/tecton-ai-ext/tecton-terraform-setup/pull/103/commits/2c6193c60517acb97f7eb69d05fcfc5d11c4bf03

**Review by jakewhlr — APPROVED** [review]
Thanks @scottames !

**Pull request merged**